cpu/riscv_common/include/periph_cpu_common.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup         cpu_riscv_common
+ * @{
+ *
+ * @file
+ * @brief           CPU specific definitions for internal peripheral handling
+ *
+ * @author          Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef PERIPH_CPU_COMMON_H
+#define PERIPH_CPU_COMMON_H
+
+#include <inttypes.h>
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Power management configuration
+ * @{
+ */
+#define PROVIDES_PM_SET_LOWEST
+/** @} */
+
+/**
+ * @brief Convenience typedef for interrupt lines
+ */
+typedef unsigned irqn;
```

Comment:
In RIOT the type are usually suffixed with `_t`. Better use `irqn_t` I think:

```suggestion
typedef unsigned irqn_t;
```

Reply:
I agree. Unrelated to this PR though.

Reply:
> Unrelated to this PR though.

Are you sure ? This change is introduced by this PR (no existing code reported by `git grep "irqn" cpu/fe310` in master)

Reply:
> Are you sure ?

Not anymore, maybe I was too quick. I'll check it and fix if I indeed introduce it here.

Reply:
Technically it is a rename from `plic_source`. I'll append the `_t` to the typedef.

Reply:
Resolved!